crosstool/generate-modulemap.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(xcode-select -p)"
+
+echo 'module "crosstool" [system] {'
```

Comment:
Do you know if quotes are needed here? The examples over at: https://clang.llvm.org/docs/Modules.html#module-maps don't use quotes and there was some discussion in the slack recently about this causing issues.

Reply:
doesn't seem to make a difference, I was just doing what bazel is doing for linux:

https://github.com/bazelbuild/bazel/blob/5ebdbfce9134f67bae664b900b5d53b215a4fcce/tools/cpp/generate_system_module_map.sh#L18

we can revisit if it breaks someone, I assume that thread is different somehow